README.md

```diff
@@ -1 +1,134 @@
+Xen Troops distro and `moulin` project file to build it. Layers in this
+repository provide final recipes to build meta-xt-prod-aos-rcar
+distro. This distro is the Aos product for Renesas boards.
+
+Those layers *may* be added and used manually, but they were written
+with [Moulin](https://moulin.readthedocs.io/en/latest/) build system,
+as Moulin-based project files provide correct entries in local.conf
+
+# Moulin project file
+
+Work is still in progress, but right now the following features are tested and working:
+
+* Renesas Salvator-XS M3 with 8GB memory is supported
+* Renesas Salvator-X H3 with 8GB memory is supported
+* Renesas Starter Kit Premiere 8GB (H3ULCB)
+* Kingfisher with Starter Kit Premiere 8GB board
+* AosBox with Starter Kit Premiere 8GB board
+* 3 domains are being built: Linux based Dom0, DomD and DomF where DomF is main Aos domain
+* Networking in DomD and DomF
+* Network (NFS) boot for DomD and DomF
+* OP-TEE client in DomF
+* Virtualized OP-TEE build
+* ARM-TF that boots into EL2
+* SD or eMMC boot
+
+Features that are present but not tested:
+
+* Renesas Salvator-XS M3 with 4GB memory
+* Renesas Salvator-XS H3 with 8GB memory
+* Renesas Starter Kit Premiere (H3ULCB)
+* Renesas Starter Kit Pro (M3ULCB)
+
+# Building
+## Requirements
+
+1. Ubuntu 18.0+ or any other Linux distribution which is supported by Poky/OE
+2. Development packages for Yocto. Refer to [Yocto
+   manual](https://www.yoctoproject.org/docs/current/mega-manual/mega-manual.html#brief-build-system-packages).
+3. You need `Moulin` installed in your PC. Recommended way is to
+   install it for your user only: `pip3 install --user
+   git+https://github.com/xen-troops/moulin`. Make sure that your
+   `PATH` environment variable includes `${HOME}/.local/bin`.
+4. Ninja build system: `sudo apt install ninja-build` on Ubuntu
+
+## Fetching
+
+You can fetch/clone this whole repository, but you actually need only
+one file from it: `prod-aos-rcar.yaml`. During build `moulin` will
+fetch this repository again into `yocto/` directory. So, to avoid
+possible confusion, we recommend to download only `prod-aos-rcar.yaml`:
+
+```
+# curl -O https://raw.githubusercontent.com/xen-troops/meta-xt-prod-aos-rcar/master/prod-aos-rcar.yaml
+```
+
+## Building
+
+Moulin is used to generate Ninja build file: `moulin
+prod-aos-rcar.yaml`. This project provides a number
+of additional options. You can use check them with
+`--help-config` command line option:
+
+```
+# moulin prod-aos-rcar.yaml --help-config
+usage: moulin prod-aos-rcar.yaml
+       [--MACHINE {salvator-x-m3,salvator-xs-m3-2x4g,salvator-xs-h3,salvator-x-h3-4x2g,h3ulcb-4x2g-ab}]
+
+Config file description: Xen-Troops development setup for Renesas RCAR Gen3
+hardware
+
+optional arguments:
+  --MACHINE {salvator-x-m3,salvator-xs-m3-2x4g,salvator-xs-h3,salvator-x-h3-4x2g,h3ulcb-4x2g-ab}
+                        RCAR Gen3-based device
+```
+
+To build for Salvator XS M3 8GB with DomU (generic yocto-based virtual
```

Comment:
Seems like DomU need to be replaced with DomF

Reply:
Also maybe it's better to use `h3ulcb-4x2g-ab` board as an example and primary target for the build.

Reply:
done